src/shared/locales/en-US/task.js

```diff
@@ -28,61 +28,61 @@ export default {
-  'copyright-yes': 'Yes, I have',
-  'copyright-no': 'No',
-  'copyright-error-message': 'Adding task failed due to copyright issues',
-  'pause-task-success': 'Pause task "{{taskName}}" success',
-  'pause-task-fail': 'Pause task "{{taskName}}" fail',
+  'copyright-warning-message': 'The file you want to download may be copyrighted audio or video, please ensure that you have permission to access to it.',
+  'copyright-yes': 'Yes, I have permission',
+  'copyright-no': 'No, I don\'t have permission',
+  'copyright-error-message': 'Failed to add task due to copyright issue',
+  'pause-task-success': 'Successfully paused task "{{taskName}}"',
+  'pause-task-fail': 'Failed to pause task "{{taskName}}"',
   'resume-task': 'Resume Task',
-  'resume-task-success': 'Resume task "{{taskName}}" success',
-  'resume-task-fail': 'Resume task "{{taskName}}" fail',
+  'resume-task-success': 'Successfully resumed task "{{taskName}}"',
+  'resume-task-fail': 'Failed to resume task "{{taskName}}"',
   'delete-task': 'Delete Task',
   'delete-selected-tasks': 'Delete Selected Tasks',
-  'delete-task-confirm': 'Are you sure to remove the "{{taskName}}" download task?',
+  'delete-task-confirm': 'Are you sure you want to remove download task "{{taskName}}"?',
   'delete-task-label': 'Delete with Files',
-  'delete-task-success': 'Delete task "{{taskName}}" success',
-  'delete-task-fail': 'Delete task "{{taskName}}" fail',
-  'remove-task-file-fail': 'Failed to delete task file, please delete it manually',
+  'delete-task-success': 'Successfully deleted task "{{taskName}}"',
+  'delete-task-fail': 'Failed to delete task "{{taskName}}"',
+  'remove-task-file-fail': 'Failed to delete task file(s), please delete them manually',
   'remove-task-config-file-fail': 'Failed to delete task config file, please delete it manually',
   'move-task-up': 'Move Task Up',
   'move-task-down': 'Move Task Down',
-  'pause-all-task': 'Pause All Task',
-  'pause-all-task-success': 'Pause all task success',
-  'pause-all-task-fail': 'Pause all task fail',
-  'resume-all-task': 'Resume All Task',
-  'resume-all-task-success': 'Resume all task success',
-  'resume-all-task-fail': 'Resume all task fail',
+  'pause-all-task': 'Pause All Tasks',
+  'pause-all-task-success': 'Successfully paused all tasks',
+  'pause-all-task-fail': 'Failed to pause all tasks',
+  'resume-all-task': 'Resume All Tasks',
+  'resume-all-task-success': 'Successfully resumed all tasks',
+  'resume-all-task-fail': 'Failed to resume all tasks',
   'clear-recent-tasks': 'Clear Recent Tasks',
   'purge-record': 'Purge Task Record',
-  'purge-record-success': 'Purge task record success',
-  'purge-record-fail': 'Purge task record fail',
+  'purge-record-success': 'Successfully purged task records',
+  'purge-record-fail': 'Failed to purge task records',
   'refresh-list': 'Refresh Task List',
-  'no-task': 'There are no current downloads',
+  'no-task': 'No active task',
```

Comment:
The `no-task` here is not just for the active task, the original text I borrowed from Folx Go+.

I suggest: `There are no current tasks.`